modules/i18n/input-method/fcitx5.nix

```
@@ -58,6 +97,38 @@ in
       sessionSearchVariables.QT_PLUGIN_PATH = [ "${fcitx5Package}/${pkgs.qt6.qtbase.qtPluginPrefix}" ];
     };
 
+    xdg =
+      let
+        mkThemeConfig = name: attrs: {
+          dataFile = {
+            "fcitx5/themes/${name}/highlight.svg".source = lib.mkIf (
+              attrs ? highlightImage
+            ) attrs.highlightImage;
+            "fcitx5/themes/${name}/panel.svg".source = lib.mkIf (attrs ? panelImage) attrs.highlightImage;
```

```suggestion
            "fcitx5/themes/${name}/highlight.svg".source = lib.mkIf (
              attrs ? highlightImage
            ) attrs.highlightImage;
            "fcitx5/themes/${name}/panel.svg".source = lib.mkIf (attrs ? panelImage) attrs.highlightImage;
```

These need to have the source wrapped in the `mkIf`

---

example: 

```nix
      home.file.".bash_logout" = mkIf (cfg.logoutExtra != "") {
        source = writeBashScript "bash_logout" cfg.logoutExtra;
      };
```